ZenPacks/zenoss/Microsoft/Windows/migrate/RemoveWinRMServices.py

```diff
@@ -21,20 +21,30 @@
 # Zenoss Imports
 from Products.ZenModel.migrate.Migrate import Version
 from Products.ZenModel.ZenPack import ZenPackMigration
+from Products.Zuul.interfaces import ICatalogTool
 
 LOG = logging.getLogger('zen.MicrosoftWindows')
 
 
 class RemoveWinRMServices(ZenPackMigration):
-    version = Version(2, 7, 0)
+    version = Version(2, 7, 1)
 
     def migrate(self, pack):
-        org = pack.dmd.Devices.getOrganizer('/Server/Microsoft')
-        devices = org.getSubDevices()
-        device_count = len(devices)
-        if device_count:
-            LOG.info('Attempting to remove incompatible Windows Services from {} device{}.'
-                     .format(device_count, 's' if device_count > 1 else ''))
-            for device in devices:
-                device.os.removeRelation('winrmservices', suppress_events=True)
+
+        results = ICatalogTool(pack.getDmdRoot("Devices")).search(types=(
+            'ZenPacks.zenoss.Microsoft.Windows.BaseDevice.BaseDevice',
+        ))
+
+        if not results.total:
+            return
+
+        LOG.info('Attempting to remove incompatible Windows Services from {} device{}.'
```

Comment:
"Attempting to remove" seems superfluous. How about just "Removing incompatible ...".

As a general best practice I would recommend sticking with old-style string formatting as supported by logging instead of string.format for log messages. It has the advantage of only doing the string formatting when the log message would actually be emitted based on logging configuration. That makes no significant difference here, but it can make significant difference in other places, and this way we keep it consistent.